x-pack/plugins/infra/public/components/metrics_explorer/helpers/calculate_domain.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { min, max, sum, isNumber } from 'lodash';
+import { MetricsExplorerSeries } from '../../../../common/http_api/metrics_explorer';
+import { MetricsExplorerOptionsMetric } from '../../../containers/metrics_explorer/use_metrics_explorer_options';
+
+const getMin = (values: Array<number | null>) => {
+  const minValue = min(values);
+  return minValue === Infinity || minValue === -Infinity ? undefined : minValue;
```

Comment:
There is `Number.isFinite` to check for both at the same time.

Reply:
Oh cool, didn't know that 👍